app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
If I'm correct, Cognito should be handling this kind of stuff (sending emails). We shouldn't need that (everything in `/mailer/` directory)?

Reply:
We have for now removed everything mailer related.